src/test/java/org/broadinstitute/hellbender/engine/spark/AddContextDataToReadSparkUnitTest.java

```diff
@@ -34,7 +36,9 @@
     @DataProvider(name = "bases")
     public Object[][] bases() {
         List<Class<?>> classes = Arrays.asList(Read.class, SAMRecord.class);
-        JoinStrategy[] strategies = JoinStrategy.values();
+        EnumSet<JoinStrategy> joinStrategies = EnumSet.allOf(JoinStrategy.class);
+        joinStrategies.remove(JoinStrategy.OVERLAPS_PARTITIONER); // exclude overlaps partitioner, since it needs sequence dictionary not available for mock source
```

Comment:
Can you extend the mock so that `ReferenceMultiSource.getReferenceSequenceDictionary()` returns a value, so that we can have test coverage of `JoinStrategy.OVERLAPS_PARTITIONER` in this class? Eg., 

        when(mockSource.getReferenceSequenceDictionary(null)).thenReturn(myDictionary);